### ISSUE TITLE
A11y_VSCode_Integrated Terminal_Split terminal_Screenreader: Screen reader is not announcing content from Terminal Section.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode#99013]
Environment Details:
VSCode Version : 1.46.0 Insider
OS Version : Win10
Additional Details:
Screenreader: NVDA[2019.3.1], JAWS[2020.70]

GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#DesktopApp;#MAS1.3.1;#Win10;#Visual Studio Code;#NVDA;#JAWS; #CAITestPass_May2020;#FTP;#STP;

Repro Steps:
1.Open Visual studio code insider editor.
2.Using "Ctrl+`" key open the Integrated Terminals.
3)Using  "Ctrl+\" key to split the terminal
5)Open Jaws/NVDA and check whether it reads terminal content.

Steps to Reproduce:

Actual:
Screen reader does not announce the terminal content.

Expected:
Screen reader should announce all content present on screen. 

Attachment For Reference:
![99013_A11y_VSCode_Integrated Terminal_Split terminal_Screenreader Screen reader is not announcing content from Terminal Section](https://user-images.githubusercontent.com/52035395/83487630-fa6bba80-a4c8-11ea-8a80-588a4f7e3939.jpg)

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by RohitJadhav-TCS** [comment]
Verified with Turn on screen reader optimized mode. Issue is no longer reproducing. Hence closing the bug. Attached screenshot for reference. 
![99013_Closed](https://user-images.githubusercontent.com/52035395/83876817-0bc9f680-a757-11ea-843d-77e26305d05a.jpg)

**Comment by RohitJadhav-TCS** [comment]
Closing bug

**Comment by Rama8345** [comment]
**Github Tags:**#Closed;
